assets/images/Leanne Gallagher _ Portfolio.html

```diff
@@ -0,0 +1,198 @@
+<!DOCTYPE html>
+<!-- saved from url=(0038)http://127.0.0.1:5500/index.html#about -->
+<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- Link to the CSS stylesheet that contains the aesthetics of the website -->
+    
+    <title>Leanne Gallagher | Portfolio</title>
+    <!-- icon for my webpage bar -->
+    
+    <!-- This is the link to the font I chose on googlefonts - 'Poppins' -->
+    
+    
+    
+  <link rel="stylesheet" href="./Leanne Gallagher _ Portfolio_files/style.css"><link rel="icon" href="http://127.0.0.1:5500/assets/images/icon.png" type="image/x-icon"><link rel="preconnect" href="https://fonts.googleapis.com/"><link rel="preconnect" href="https://fonts.gstatic.com/" crossorigin=""><link href="./Leanne Gallagher _ Portfolio_files/css2" rel="stylesheet"></head>
+
+  <body>
+    <header class="header">
+      <img src="./Leanne Gallagher _ Portfolio_files/profile-img.png">
+      <h1>Leanne Gallagher</h1>
+      <!-- The navigation bar containing internal links to each important section of the webpage -->
+      <nav class="nav-container">
+        <a class="nav-link" href="http://127.0.0.1:5500/index.html#about">About Me</a>
+        <a class="nav-link" href="http://127.0.0.1:5500/index.html#portfolio">Portfolio</a>
+        <a class="nav-link" href="http://127.0.0.1:5500/index.html#contact">Contact</a>
```

Comment:
These do seem to work but be careful with using this method for linking. This may not work for everyone. Have a look at the first homework at how the links were done or google another way of doing this.